CSOC_Documentation.md

```diff
@@ -63,7 +63,7 @@ A VPC peering connection is used for the communication between the admin VM in t
 
   `gen3 approve_vpcpeering_request <child_vpc_name>`
 
-  The underneath script basically looks for a new VPC peering request, if there is a request in a pending state, it is accepted, tagged and  an appropriate route to the csoc_main_vpc private subnet route table is added.
+  The underneath script basically looks for a new VPC peering request, if there is a request in a pending state, it is accepted, tagged and  an appropriate route to the csoc_main_vpc private subnet route table is added. This scirpt accepts an optional flag `--get-route-table` which programtically gets the route table id(s) of the VPC which is accepting the peering request.
```

Comment:
a couple of typos:

`scrirpt` -> `script`
`programtically` ->`programmatically`

Reply:
Typos Fixed